src/pages/signin/SignInPage.js

```diff
@@ -37,6 +43,15 @@ const propTypes = {
 
         /** Is this account having trouble receiving emails */
         hasEmailDeliveryFailure: PropTypes.bool,
+
+        /** Whether the account is in a loading state */
```

Comment:
I think this comment used above is better
`/** Whether or not a sign on form is loading (being submitted) */`